lib/java_buildpack/container/tomee/tomee_instance.rb

```diff
@@ -34,6 +44,21 @@ def tomcat_7_compatible
         @version < TOMEE_7
       end
 
+      private
+
+      def web_inf_lib
+        if ear?
+          # copy additional libraries to tomee lib folder
+          tomcat_lib
```

Comment:
Update PR @violetagg, @kelapure to support point 1 above - currently it blindly links the jars in `drivers` folder to tomee/lib, can you please review if you get a chance